pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -1913,40 +1913,43 @@ class HdRprApiImpl {
             m_dirtyFlags |= ChangeTracker::DirtyScene;
         }
 
-        if (m_rprContextMetadata.pluginType == kPluginNorthstar) {
```

Comment:
I remove condition `m_rprContextMetadata.pluginType == kPluginNorthstar` because it does not needed after removing Tahoe